vendor/knative.dev/eventing/pkg/apis/sources/v1alpha1/ping_defaults.go

```diff
@@ -18,20 +18,12 @@ package v1alpha1
 
 import (
 	"context"
-
-	eventingduckv1alpha1 "knative.dev/eventing/pkg/apis/duck/v1alpha1"
 )
 
-func (s *Parallel) SetDefaults(ctx context.Context) {
-	if s != nil && s.Spec.ChannelTemplate == nil {
-		// The singleton may not have been set, if so ignore it and validation will reject the
-		// Channel.
-		if cd := eventingduckv1alpha1.ChannelDefaulterSingleton; cd != nil {
-			channelTemplate := cd.GetDefault(s.Namespace)
-			s.Spec.ChannelTemplate = channelTemplate
-		}
-	}
+func (s *PingSource) SetDefaults(ctx context.Context) {
 	s.Spec.SetDefaults(ctx)
 }
 
-func (ss *ParallelSpec) SetDefaults(ctx context.Context) {}
+func (ss *PingSourceSpec) SetDefaults(ctx context.Context) {
+	// TODO anything?
```

Comment:
How about clarifying with the author instead of merging this? We all know most TODOs stay in the code for ever 😉